types/tests/schema-test.ts

```diff
@@ -21,7 +21,7 @@ schema.create("foo", { x: true }); // $ExpectError
 schema.create("cow"); // $ExpectError
 
 schema.find("foo", "123"); // $ExpectType ModelInstance<FlattenRelationships<Assign<{}, { attr: string; }>> & FlattenFactoryMethods<Assign<{}, { attr: string; }>>> | null
-schema.find("foo", ["123"]).models[0]; // $ExpectType ModelInstance<FlattenRelationships<Assign<{}, { attr: string; }>> & FlattenFactoryMethods<Assign<{}, { attr: string; }>>>
+schema.find("foo", ["123"]).models[0]; // $ExpectType ["foo"]
```

Comment:
@jamescdavis is this expected type right?

Reply:
Nope! See my review below.